src/java.base/share/classes/java/lang/doc-files/threadPrimitiveDeprecation.html

```diff
@@ -71,7 +71,7 @@ <h2>What should I use instead of <code>Thread.stop</code>?</h2>
 communication of the stop-request, the variable must be
 <code>volatile</code> (or access to the variable must be
 synchronized).</p>
-<p>For example, suppose your applet contains the following
+<p>For example, suppose your application contains the following
```

Comment:
The example below is still applet specific (e.g., `repaint()`). May need to be more generic.

Reply:
`repaint()` could be substituted with another arbitrary method name to highlight some work this thread has to perform.

given the thread name `blink`, perhaps we could just use `blink()` in this example.

Reply:
Don't waste too much time on the "Java Thread Primitive Deprecation" page. The last remnant is the no-arg Thread.stop method which throws UOE unconditionally. Once that is removed then the doc-file pages will be removed too.

Reply:
Kept it simple and simply replaced with Sean's suggestion of `blink()`.